features/doc-access-collections.feature

```diff
@@ -27,3 +27,9 @@ Feature: Access document collections
   Scenario: Access the tables collection of a document
     Given a document having three tables
      Then document.tables is a list containing three tables
+
+
+  @wip
+  Scenario: Document.bookmarks
+    Given a Document object as document
+     Then document.bookmarks is a Bookmarks object
```

Comment:
The project's style for acceptance tests has evolved somewhat since many of these were written. This one reflects the current style, which adopts the perspective that the "user" in these user acceptance tests is a _developer_ using the _API_. So the scenario name is usually terse like this, often {object}.{method or property}, sometimes with "getter" or "setter" appended when it's two aspects of a property.

The given statement in this style mentions the "variable" name used for the object in an "as" clause at the end.

This test is quite simplified from your original version. This reflects that there are three "levels" or "steps" that are tested, each individually:

1. A document object has a `.bookmarks` property that returns (unconditionally and idempotently in this case) a `Bookmarks` object.

2. A Bookmarks object has certain behaviors (len, indexed access, interable, etc.)

3. An individual _Bookmark_ object has certain behaviors.

These are each tested separately and not mixed in one test.